### PR TITLE
preemeijer w3m cheat sheet Goodie

### PR DESCRIPTION
**Pull request by preemeijer**
**What does your Instant Answer do?**
Cheat sheet for w3m text based browser

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://w3m.sourceforge.net/MANUAL#Key:orig

**Why did you choose this data source?**
Original page

**Are there any other alternative (better) data sources?**

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Linux users

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes

**Are you having any problems? Do you need our help with anything?**
Had a problem with making a screenshot but restarting Codio helped

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot-w3m-cheat-sheet

-----
IA Page: https://duck.co/ia/view/w3m_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/w3m.json [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+        "Miscellany",
+        "Line-edit mode"
+    ],
+    "sections": {
+        "Page/Cursor motion": [{
+            "val": "Forward page",
+            "key": "[SPC], [C-v]"
+        }, {
+            "val": "Backward page",
+            "key": "[b], [ESC] + [v]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/w3m.json]: @preemeijer you can drop the `+` in these cases. If there isn't a `,` between the keys, it's implied that they're combined :+1:

File: share/goodie/cheat_sheets/w3m.json [hunk 2]
```diff
@@ -0,0 +1,297 @@
+{
+    "id": "w3m_cheat_sheet",
+    "name": "w3m",
+    "description": "Text based Web browser",
+    "metadata": {
+        "sourceName": "w3mCheatSheet",
+        "sourceUrl": "http://w3m.sourceforge.net/MANUAL#Key:orig"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/w3m.json]: Do they support HTTPS?

Comment (preemeijer) [inline share/goodie/cheat_sheets/w3m.json]: I've tried to place HTTPS but unfortunately I get the following error: Unable to connect.

File: share/goodie/cheat_sheets/w3m.json (hunk 1 above)
Comment (preemeijer) [inline share/goodie/cheat_sheets/w3m.json]: Ok, so I remove all the + in this cheat sheet?

Comment (mintsoft) [inline share/goodie/cheat_sheets/w3m.json]: @preemeijer Exactly

**Comment by preemeijer** [comment]
Do I need to tell or do something else to get this cheat sheet active or reviewed again after my changes?

Greetings, Paul

**Comment by mintsoft** [comment]
@preemeijer nothing especially, however commits alone don't trigger notifications so usually we ask people to comment when they've updated so we're informed

**Comment by preemeijer** [comment]
@mintsoft Okay. Thanks for telling me and done the thing that was needed then ;)
I will wait if the cheat sheet is good to go or that I need to correct something else.

**Comment by moollaza** [comment]
@preemeijer sorry for the delay! and thanks for the updates :+1: 

This LGTM :+1:

**Comment by moollaza** [comment]
@preemeijer congrats! It's live: https://duckduckgo.com/?q=w3m+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by preemeijer** [comment]
@moollaza I saw that it is live and shared it with others and tomorrow we will go to the next one(s) with the global Quack and Hack :+1:

**Pull request merged**